### PR TITLE
Add "concat" method

### PR DESCRIPTION
**Pull request by binkpitch**
Add "concat" method as described in #4.

### CODE REVIEW

File: src/set.ts [hunk 1]
```diff
@@ -217,4 +217,23 @@ export class SuperchargedSet<T> implements Iterable<T> {
...
+
+  /**
+   * Appends values to the end of the array.
+   *
+   * @param {*} values
+   *
+   * @returns {SuperchargedSet}
+   */
+  concat (...values: Array<T | T[]>): this {
+    for (const value of values) {
```
Comment (marcuspoehls) [inline src/set.ts:229]: The set already has a `.toArray()` method creating an array of the set’s entries. You may then combine it with the array’s concat method, like `this.toArray().concat(...values)`. What do you think?

**Review by marcuspoehls — COMMENTED** [review]
Hey, I left a comment related to use existing array functionality to concat values onto the set. You may need to create a new set instance when leveraging the array methods. What do you think?

**Comment by marcuspoehls** [comment]
@binkpitch Merging the pull request for the `.count()` method introduced merge errors in the repository. Can you please resolve them for this pull request? Sorry for the extra effort

File: src/set.ts (hunk 1 above)
Comment (binkpitch) [reply src/set.ts]: I like the `.toArray()` way and have updated the code.

**Comment by marcuspoehls** [comment]
Looks good now. Thank you very much!

**Pull request merged**